### DISCUSSION TITLE
Wrong Device Mapping WHD02

### DISCUSSION THREAD

**Discussion by ZiGy67** [Koenkk/zigbee2mqtt#14808] [Devices]
I have an AUBESS Power Monitor Switch. This device is recognized as WHD02 since Z2M version 1.28.1-1.
The WHD02 is a device that does not support power monitoring.

I tried to make a custom configuration in Z2M but the device is always recognized as WHD02.

How can I correct this in Z2M?

![Aubess Power Monitoring Switch](https://user-images.githubusercontent.com/70607836/199982020-3c79a7a1-56a9-41a5-924b-79b62fa4b2cc.png)

![WHD02](https://user-images.githubusercontent.com/70607836/199982212-dcfa9ebd-313b-488c-8d8c-c6c025f9318f.jpg)

Logging: 

warn  2022-11-04 11:58:21: Device '0xa4c138ccef50f472' left the network
info  2022-11-04 11:58:21: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c138ccef50f472","ieee_address":"0xa4c138ccef50f472"},"type":"device_leave"}'
info  2022-11-04 11:58:21: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"left_network","meta":{"friendly_name":"0xa4c138ccef50f472"},"type":"device_removed"}'
info  2022-11-04 11:58:27: Device '0xa4c138ccef50f472' joined
info  2022-11-04 11:58:27: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c138ccef50f472","ieee_address":"0xa4c138ccef50f472"},"type":"device_joined"}'
info  2022-11-04 11:58:27: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0xa4c138ccef50f472"},"type":"device_connected"}'
info  2022-11-04 11:58:27: Starting interview of '0xa4c138ccef50f472'
info  2022-11-04 11:58:27: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c138ccef50f472","ieee_address":"0xa4c138ccef50f472","status":"started"},"type":"device_interview"}'
info  2022-11-04 11:58:27: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0xa4c138ccef50f472"},"type":"pairing"}'
debug 2022-11-04 11:58:27: Device '0xa4c138ccef50f472' announced itself
info  2022-11-04 11:58:27: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xa4c138ccef50f472","ieee_address":"0xa4c138ccef50f472"},"type":"device_announce"}'
info  2022-11-04 11:58:27: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0xa4c138ccef50f472"},"type":"device_announced"}'
debug 2022-11-04 11:58:27: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"tuyaBacklightMode":1}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:27: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:27: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"moesStartUpOnOff":2}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:27: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:27: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"tuyaBacklightMode":1}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:27: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:27: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"tuyaBacklightMode":1}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:27: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"rmsVoltage":233}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'seMetering', data '{"currentSummDelivered":[0,0]}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"rmsVoltage":233}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"modelId":"TS000F"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"_TZ3000_xkap8wtb"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"powerSource":1}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"zclVersion":3}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"appVersion":65}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"stackVersion":2}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:28: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:28: Skipping message, definition is undefined and still interviewing
debug 2022-11-04 11:58:29: Received Zigbee message from '0xa4c138ccef50f472', type 'readResponse', cluster 'genBasic', data '{"swBuildId":"0122052017"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:29: Skipping message, definition is undefined and still interviewing
info  2022-11-04 11:58:29: Successfully interviewed '0xa4c138ccef50f472', device has successfully been paired
info  2022-11-04 11:58:29: Device '0xa4c138ccef50f472' is supported, identified as: TuYa Wall switch module (WHD02)
info  2022-11-04 11:58:29: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"Wall switch module","exposes":[{"features":[{"access":7,"description":"On/off state of the switch","name":"state","property":"state","type":"binary","value_off":"OFF","value_on":"ON","value_toggle":"TOGGLE"}],"type":"switch"},{"access":7,"description":"Controls the behavior when the device is powered on","name":"power_on_behavior","property":"power_on_behavior","type":"enum","values":["off","previous","on"]},{"access":7,"description":"Switch type settings","name":"switch_type","property":"switch_type","type":"enum","values":["toggle","state","momentary"]},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"WHD02","options":[],"supports_ota":false,"vendor":"TuYa"},"friendly_name":"0xa4c138ccef50f472","ieee_address":"0xa4c138ccef50f472","status":"successful","supported":true},"type":"device_interview"}'
info  2022-11-04 11:58:29: Configuring '0xa4c138ccef50f472'
info  2022-11-04 11:58:29: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"description":"Wall switch module","friendly_name":"0xa4c138ccef50f472","model":"WHD02","supported":true,"vendor":"TuYa"},"type":"pairing"}'
info  2022-11-04 11:58:29: MQTT publish: topic 'homeassistant/switch/0xa4c138ccef50f472/switch/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c138ccef50f472/set","device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472","payload_off":"OFF","payload_on":"ON","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_switch_zigbee2mqtt","value_template":"{{ value_json.state }}"}'
info  2022-11-04 11:58:29: MQTT publish: topic 'homeassistant/sensor/0xa4c138ccef50f472/power_on_behavior/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"enabled_by_default":false,"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 power on behavior","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
info  2022-11-04 11:58:29: MQTT publish: topic 'homeassistant/select/0xa4c138ccef50f472/power_on_behavior/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c138ccef50f472/set/power_on_behavior","device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 power on behavior","options":["off","previous","on"],"state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
info  2022-11-04 11:58:29: MQTT publish: topic 'homeassistant/sensor/0xa4c138ccef50f472/switch_type/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"enabled_by_default":false,"entity_category":"config","icon":"mdi:tune","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 switch type","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_switch_type_zigbee2mqtt","value_template":"{{ value_json.switch_type }}"}'
info  2022-11-04 11:58:29: MQTT publish: topic 'homeassistant/select/0xa4c138ccef50f472/switch_type/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c138ccef50f472/set/switch_type","device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"entity_category":"config","icon":"mdi:tune","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 switch type","options":["toggle","state","momentary"],"state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_switch_type_zigbee2mqtt","value_template":"{{ value_json.switch_type }}"}'
info  2022-11-04 11:58:29: MQTT publish: topic 'homeassistant/sensor/0xa4c138ccef50f472/linkquality/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
debug 2022-11-04 11:58:29: Received MQTT message on 'homeassistant/switch/0xa4c138ccef50f472/switch/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c138ccef50f472/set","device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472","payload_off":"OFF","payload_on":"ON","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_switch_zigbee2mqtt","value_template":"{{ value_json.state }}"}'
debug 2022-11-04 11:58:29: Received MQTT message on 'homeassistant/sensor/0xa4c138ccef50f472/power_on_behavior/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"enabled_by_default":false,"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 power on behavior","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
debug 2022-11-04 11:58:29: Received MQTT message on 'homeassistant/select/0xa4c138ccef50f472/power_on_behavior/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c138ccef50f472/set/power_on_behavior","device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 power on behavior","options":["off","previous","on"],"state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
debug 2022-11-04 11:58:29: Received MQTT message on 'homeassistant/sensor/0xa4c138ccef50f472/switch_type/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"enabled_by_default":false,"entity_category":"config","icon":"mdi:tune","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 switch type","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_switch_type_zigbee2mqtt","value_template":"{{ value_json.switch_type }}"}'
debug 2022-11-04 11:58:29: Received MQTT message on 'homeassistant/select/0xa4c138ccef50f472/switch_type/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0xa4c138ccef50f472/set/switch_type","device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"entity_category":"config","icon":"mdi:tune","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 switch type","options":["toggle","state","momentary"],"state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_switch_type_zigbee2mqtt","value_template":"{{ value_json.switch_type }}"}'
debug 2022-11-04 11:58:29: Received MQTT message on 'homeassistant/sensor/0xa4c138ccef50f472/linkquality/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0xa4c138ccef50f472"],"manufacturer":"TuYa","model":"Wall switch module (WHD02)","name":"0xa4c138ccef50f472","sw_version":"0122052017"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0xa4c138ccef50f472","name":"0xa4c138ccef50f472 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0xa4c138ccef50f472","unique_id":"0xa4c138ccef50f472_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
info  2022-11-04 11:58:29: Successfully configured '0xa4c138ccef50f472'
debug 2022-11-04 11:58:30: Received Zigbee message from '0xa4c138ccef50f472', type 'read', cluster 'genTime', data '["localTime"]' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:43: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info  2022-11-04 11:58:43: MQTT publish: topic 'zigbee2mqtt/0xa4c138ccef50f472', payload '{"linkquality":87,"power_on_behavior":null,"state":"OFF","switch_type":null}'
debug 2022-11-04 11:58:43: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"32768":0}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:44: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"tuyaBacklightMode":1}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:44: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"moesStartUpOnOff":2}' from endpoint 1 with groupID 0
info  2022-11-04 11:58:44: MQTT publish: topic 'zigbee2mqtt/0xa4c138ccef50f472', payload '{"linkquality":87,"power_on_behavior":"previous","state":"OFF","switch_type":null}'
debug 2022-11-04 11:58:44: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"rmsVoltage":233}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:44: No converter available for 'WHD02' with cluster 'haElectricalMeasurement' and type 'attributeReport' and data '{"rmsVoltage":233}'
debug 2022-11-04 11:58:44: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"rmsCurrent":83}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:44: No converter available for 'WHD02' with cluster 'haElectricalMeasurement' and type 'attributeReport' and data '{"rmsCurrent":83}'
debug 2022-11-04 11:58:44: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'haElectricalMeasurement', data '{"activePower":0}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:44: No converter available for 'WHD02' with cluster 'haElectricalMeasurement' and type 'attributeReport' and data '{"activePower":0}'
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'seMetering', data '{"currentSummDelivered":[0,0]}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:45: No converter available for 'WHD02' with cluster 'seMetering' and type 'attributeReport' and data '{"currentSummDelivered":[0,0]}'
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'manuSpecificTuya_3', data '{"switchType":0}' from endpoint 1 with groupID 0
info  2022-11-04 11:58:45: MQTT publish: topic 'zigbee2mqtt/0xa4c138ccef50f472', payload '{"linkquality":83,"power_on_behavior":"previous","state":"OFF","switch_type":"toggle"}'
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,83,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:45: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,83,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,84,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:45: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,84,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,85,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:45: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,85,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'read', cluster 'genTime', data '["localTime"]' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:45: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,87,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:45: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,87,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:46: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,88,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:46: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,88,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:46: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,89,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:46: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,89,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:46: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,90,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:46: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,90,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:46: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,91,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:46: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,91,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:46: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,92,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:46: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,92,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:46: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,93,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:46: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,93,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:47: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,94,10,1,208,72,2,0,6],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:47: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,94,10,1,208,72,2,0,6],"type":"Buffer"}'
debug 2022-11-04 11:58:47: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,95,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:47: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,95,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:47: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,96,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:47: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,96,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:47: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,97,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:47: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,97,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:47: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,98,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:47: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,98,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:48: Received Zigbee message from '0xa4c138ccef50f472', type 'raw', cluster '57344', data '{"data":[8,99,10,2,208,72,2,0,10],"type":"Buffer"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:48: No converter available for 'WHD02' with cluster '57344' and type 'raw' and data '{"data":[8,99,10,2,208,72,2,0,10],"type":"Buffer"}'
debug 2022-11-04 11:58:48: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster '57344', data '{"53251":"AAAA"}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:48: No converter available for 'WHD02' with cluster '57344' and type 'attributeReport' and data '{"53251":"AAAA"}'
debug 2022-11-04 11:58:48: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"onTime":0}' from endpoint 1 with groupID 0
debug 2022-11-04 11:58:48: Received Zigbee message from '0xa4c138ccef50f472', type 'attributeReport', cluster 'genOnOff', data '{"offWaitTime":0}' from endpoint 1 with groupID 0

**Comment by ZiGy67** [comment]
Any Coments?

**Comment by Koenkk** [comment]
This issue has been fixed in the dev branch (power monitoring), will be detected as `TS000F_power` (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by ZiGy67** [comment]
will this option be added to an upcomming release ?

**Reply by Koenkk** [reply]
Yes,on  1 December.

**Reply by ZiGy67** [reply]
Thanks !
